ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ContractKeysIT.scala

```diff
@@ -195,7 +195,7 @@ final class ContractKeysIT extends LedgerTestSuite {
       assertGrpcError(
         duplicateKeyFailure,
         Status.Code.ABORTED,
-        Some(Pattern.compile("Inconsistent")),
+        Some(Pattern.compile("DuplicateKeys: at least one contract key is not unique")),
```

Comment:
Sounds like we need to keep this particular assertion. You can fix the error by changing the message in `ActiveLedgerStateManager` instead.

Reply:
No, wait, I misunderstood. I think this is a legit change. My solution was correct though: you can fix the failure in Sandbox Classic by changing the message in `ActiveLedgerStateManager`.

Reply:
This message, in particular, comes from `ExternallyInconsistentTransaction.DuplicateKeys`, I assume that one we need to change to `Inconsistent: DuplicateKey: contract key is not unique`

Reply:
I guess whatever it was before?